allure-report-face/src/variables.css

```diff
@@ -1,10 +1,10 @@
 $brand-blue: #0044bb;
 $brand-red: #ff0000;
 
-$color-passed: #3bbd25;
-$color-failed: #ff0000;
-$color-broken: #ffa924;
-$color-canceled: #999999;
+$color-passed: #97cc64;
+$color-failed: #fd5a3e;
+$color-broken: #ffd963;
```

Comment:
Better, but I still don't understand the point of changing this. Why can you not just leave current colors as it is?

Okay, let's change red color as it is so bright, but what the reason of changing others? That made all of them less readable